lasy/utils/laser_utils.py

```diff
@@ -4,6 +4,14 @@
 from scipy.signal import hilbert
 from skimage.restoration import unwrap_phase
 
+try:
+    from axiprop.lib import PropagatorFFT2, PropagatorResampling
+    from axiprop.containers import ScalarFieldEnvelope
+
+    axiprop_installed = True
+except ImportError:
+    axiprop_installed = False
+
```

Comment:
Should we wait until a decision about #164 is made to add these checks?

Reply:
True, I just removed the checks. We'll make it consistent with #164 when a decision is made.